src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -780,6 +785,37 @@ void Transformations::PostLpt() {
     CPU_REGISTER_PASS_X64(postLPTPassManager, ov::pass::RoPEFusion);
     CPU_REGISTER_PASS_X64(postLPTPassManager, CausalMaskPreprocessFusion);
 
+    // MLP & QKV fusion optimizations is focused on throughput, only enabled on AMX-bf16 & LLM serving use cases.
+    auto can_use_amx_bf16 = dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core_amx) && (inferencePrecision == element::bf16);
+    if (can_use_amx_bf16) {
+        auto has_paged_attention = op::util::has_op_with_type<ov::op::PagedAttentionExtension>(model);
+        if (has_paged_attention) {
+            CPU_REGISTER_PASS_X64(postLPTPassManager, MLPFusion);
+            CPU_SET_CALLBACK_X64(postLPTPassManager,
+                [this](const_node_ptr &node) -> bool {
+                    std::string errorMsg;
+                    return node::LLMMLP::isSupportedOperation(node, errorMsg);
```

Comment:
minor: would propose to add avaialble isa check inside `isSupportedOperation` methods as well.

Reply:
kept unchanged.